src/Form/matching.js

```diff
@@ -161,40 +180,43 @@ class Matching {
      * Tries to infer the input type for an input
      *
      * @param {HTMLInputElement|HTMLSelectElement} input
-     * @param {HTMLFormElement} formEl
+     * @param {HTMLElement} formEl
      * @param {{isLogin?: boolean}} [opts]
      * @returns {SupportedTypes}
      */
     inferInputType (input, formEl, opts = {}) {
         const presetType = getInputType(input)
-        if (presetType !== 'unknown') return presetType
+        if (presetType !== 'unknown') {
+            return presetType
+        }
 
-        // For CC forms we run aggressive matches, so we want to make sure we only
-        // run them on actual CC forms to avoid false positives and expensive loops
+        this.setActiveElementStrings(input, formEl)
+
+        // // For CC forms we run aggressive matches, so we want to make sure we only
+        // // run them on actual CC forms to avoid false positives and expensive loops
         if (this.isCCForm(formEl)) {
```

Comment:
No action item right now, but I think at this point this `isCCForm` is unnecessary. We already run those aggressive checks for identities anyway, so this `isCCForm` could be useless and even harmful if it returns a false negative because it stops the more comprehensive checks (something I've seen in the wild).